src/css/abstracts/variables.css

```diff
@@ -12,6 +12,7 @@
 	--beauty-bush: #efb5ba;
 	--saddle-brown: #564000;
 	--double-colonial-white: #efdfab;
+	--san-felix: #096915;
```

Comment:
Don't keep the color name as css variable as it doesn't make sense. Keep the shades of colours instead like blue 100 , blue 200 and similar as described in the Design system guide on lms

Reply:
Thanks Farhan for the suggestion, changed it accordingly